login_server/src/cpp/SingletonManager/EmailManager.cpp

```diff
@@ -141,6 +141,33 @@ int EmailManager::ThreadFunction()
 				mailClientSession.sendMessage(mailMessage);
 					email_sended = true;
 				}
+				catch (Poco::Net::SSLConnectionUnexpectedlyClosedException& ex) {
+					// it is a mad idea to send an email if the email sending failed
+					// errors.sendErrorsAsEmail();
+					// better wait instead and try again, it seems that the strato mail server sometimes discard or connection
+					// MAGIC NUMBER: sleep time if email sending failed (ssl connection was discarded from mailserver)
+					// wait 5 minute for the next try
+					Poco::Thread::sleep(5 * 60 * 1000);
```

Comment:
I would define a constant here `EMAIL_SEND_RETRY_TIME` or alike. I think 5 minutes with 4 retries is quite alot for a connection issue.